manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -83,12 +66,7 @@ spec:
         summary: etcd cluster 99th percentile commit durations are too high.
       expr: |
         histogram_quantile(0.99, rate(etcd_disk_backend_commit_duration_seconds_bucket{job=~".*etcd.*"}[5m]))
-        > on () group_left (type)
```

Comment:
This was made in a previous PR manually against the generated yaml. Letting it revert per convo on Slack.